resources/views/users/dashboard.blade.php

```diff
@@ -1,61 +1,113 @@
+              <button type="submit" class="btn btn-danger">X</button>
+            </form>
+          </div>
+          @endif
+        </div>
+        <small>By: {{ $comment->user_name }}</small>
+        <small>{{ $comment->created_at }}</small> @if($comment->created_at != $comment->updated_at)
+        <small>Edited at:{{ $comment->updated_at }}</small> @endif @if (!$comment->approved)
+        <p style="background-color:rgb(255,80,80); color: white;">Your comment is pending approval by a moderator. Until it is approved, only you will be able to see it.</p>
         @endif
-      </li>
+      </div>
       @endforeach @else
-      <p>You haven't made any reviews yet</p>
+      <p>You haven't made any comments yet</p>
       @endif
-    </ul>
-    <hr />
-    <h2>My Comments</h2>
-    @if (count($comments) > 0) @foreach($comments as $comment)
-    <div class="card mb-2 bg-light text-dark p-2">
-      <p>{{ $comment->content }}</p>
-      <p>By: {{ $comment->user_name }}</p>
-      <p>{{ $comment->created_at }}</p>
-      @if($comment->created_at != $comment->updated_at)
-      <p>Edited at:{{ $comment->updated_at }}</p>
-      @endif
-    </div>
+      </ul>
+      <hr />
+      <h2>My Comments</h2>
+      @if (count($comments) > 0) @foreach($comments as $comment)
+      <div class="card mb-2 bg-light text-dark p-2">
+        <p>{{ $comment->content }}</p>
+        <p>By: {{ $comment->user_name }}</p>
+        <p>{{ $comment->created_at }}</p>
+        @if($comment->created_at != $comment->updated_at)
+        <p>Edited at:{{ $comment->updated_at }}</p>
+        @endif
+      </div>
       @if (!$comment->approved)
-        <p>Your comment is pending approval by a moderator.
-        Until it is approved, only you will be able to see it.</p>
+      <p>Your comment is pending approval by a moderator. Until it is approved, only you will be able to see it.</p>
+      @endif @endforeach @else
+      <p>You haven't made any comments yet</p>
       @endif
-    @endforeach @else
-    <p>You haven't made any comments yet</p>
-    @endif
-    </ul>
+      </ul>
     </div>
     @if ($administrate_reviews && $administrate_comments)
     <div class="right-container">
-      <a href="admin/reviews"><h2>Administrate Reviews</h2></a>
-      <a href="admin/comments"><h2>Administrate Comments</h2></a>
+      <a href="admin/reviews">
+        <h2>Administrate Reviews</h2>
+      </a>
+      <a href="admin/comments">
+        <h2>Administrate Comments</h2>
+      </a>
     </div>
     @endif
   </div>
 </div>
+
+<script>
+  const commentToDelete = $('.delete-comment');
+     function deleteComment (event) {
+        event.preventDefault();
+        $.ajax(
+          {
+          url: `/comments/${event.target[2].value}`,
+          method: 'DELETE',
+          headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+          }
+          statusCode: {
+    200: function() {
```

Comment:
Made for testing purposes I assume, plaz remove

Reply:
:+1: